### PR TITLE
OC-4529 Remove the "Verify your Identity" banner on ecommerce receipt page

### PR DESCRIPTION
**Pull request by lgp171188**
Remove the "Verify your Identity" banner on ecommerce receipt page and replace it with a button to go to the LMS dashboard.

**Screenshot**
receipt_page_no_verification_block

**Testing instructions**
- Revert the changes in PR #3.
- Create a course on LMS.
- Add the course on the ecommerce portal under one of the paid options.
- Generate a '100% off' enrollment coupon for the course.
- Enroll a student using the enrollment coupon created in the previous step.
- Check that the 'Verify your Identity' banner does not show up and instead there is a button to navigate to the LMS dashboard.

**Reviewers**
- [ ] @kaizoku

### CODE REVIEW

File: ecommerce/templates/edx/checkout/receipt.html [hunk 1]
```diff
@@ -151,8 +151,12 @@ <h2>{% trans "Order Information" %}</h2>
       </div>
 
       {% if verification_url and not user_verified %}
```
Comment (kaizoku) [inline ecommerce/templates/edx/checkout/receipt.html]: Would it be better to just remove this if/else clause altogether?

The text and styling for the button added here is slightly different from the one below, if we didn't duplicate them we wouldn't have to try to match them.

**Review by kaizoku — CHANGES_REQUESTED** [review]

File: ecommerce/templates/edx/checkout/receipt.html (hunk 1 above)
Comment (lgp171188) [reply ecommerce/templates/edx/checkout/receipt.html]: @kaizoku, that was so obvious and I am embarrassed to have missed it. Will fix it and update the PR now.

**Comment by lgp171188** [comment]
@kaizoku, I have updated the PR to address your review comment.

**Comment by kaizoku** [comment]
Thanks for the update @lgp171188 
:+1:

- [✓] I tested this: (deployed in devstack, checked that verification banner doesn't show)
- [✓] I read through the code
- [✓] I checked for accessibility issues
- [n/a] Includes documentation

**Review by kaizoku — APPROVED** [review]

**Comment by kaizoku** [comment]
Thanks for the update @lgp171188 
:+1:

- [✓] I tested this: (deployed in devstack, checked that verification banner doesn't show)
- [✓] I read through the code
- [✓] I checked for accessibility issues
- [n/a] Includes documentation

**Comment by lgp171188** [comment]
@kaizoku, I am merging this PR now.

**Pull request merged**